src/assets/styles/mixins/breakpoints.scss

```diff
@@ -1,10 +1,19 @@
+// Global Breakpoints
+$breakpoint-laptop: 1024px;
+$breakpoint-tablet: 767px;
+$breakpoint-min-mobile: 320px;
```

Comment:
I think cool kids these days try to avoid `mobile` and `tablet` in names, though I don't have any strong preference about it.

Reply:
I used those names because that is what the Chrome developer tools call them. I can change them if you like.

<img width="889" alt="image" src="https://user-images.githubusercontent.com/29865743/184140944-4cadcb51-9ff2-437f-b18d-4685b63c6539.png">

<img width="889" alt="image" src="https://user-images.githubusercontent.com/29865743/184141040-a782cb0c-27fb-41a0-88d1-a837adbd3c15.png">